testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/validation/CustomExceptionMapperTest.java

```diff
@@ -64,6 +65,12 @@
                         CustomExceptionMapperReport.class)
                 .addAsLibrary(file, "validation-versioning.jar")
                 .addAsWebInfResource(EmptyAsset.INSTANCE, "beans.xml");
+
+        war.addAsManifestResource(PermissionUtil.createPermissionsXmlAsset(
+            new RuntimePermission("getClassLoader"),
```

Comment:
Only `new RuntimePermission("accessDeclaredMembers")` is required to make the test pass, no getClassloader permission is required.

The stacktrace of the AccessControll exception (attached in https://issues.jboss.org/browse/JBEAP-10058) shows that problem is in the class: `org.jboss.resteasy.plugins.validation.ConstraintTypeUtil11.getMethod(ConstraintTypeUtil11.java:95)`.

Why is it better in this case to give the 'accessDeclaredMembers' permission to the application, instead of creating AccessController.doPrivileged() block in the resteasy code?

 For example when application itself calls some code which reguires permission directly:
`Thread.currentThread().getContextClassLoader()`, it is obvious it must have granted the permission directly.
If the above getContextClassLoader() is called somewhere in the resteasy classes then probably it should be handled in the resteasy code.